nvflare/private/fed/server/job_cmds.py

```diff
@@ -111,7 +111,9 @@ def list_jobs(self, conn: Connection, args: List[str]):
                     conn.append_error("No jobs matching the searching criteria")
                     return
 
-                if len(filtered_jobs) == 1 or parsed_args.d:
+                filtered_jobs.sort(key=lambda job: job.meta.get("submit_time", 0.0))
```

Comment:
for "submit_time", might want to use the constant variable defined in job_def.py?

Reply:
Done